src/sdam/server.ts

```diff
@@ -108,24 +118,23 @@ export class Server extends TypedEventEmitter<ServerEvents> {
   /** @internal */
   s: ServerPrivate;
   serverApi?: ServerApi;
-  clusterTime?: ClusterTime;
   ismaster?: Document;
   [kMonitor]: Monitor;
 
   /** @event */
-  static readonly SERVER_HEARTBEAT_STARTED = 'serverHeartbeatStarted' as const;
+  static readonly SERVER_HEARTBEAT_STARTED = SERVER_HEARTBEAT_STARTED;
```

Comment:
Same on the constants front here. Maybe this is a new ticket to clean this up and import directly everywhere?